dev/com.ibm.ws.transport.http/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -88,9 +88,9 @@
             <Option label="%onError.ignore" value="IGNORE"/>   
         </AD>
 
-        <AD id="httpOptionsRef" name="%endpoint.httpOptions" description="%endpoint.httpOptions.desc"
-            ibm:reference="com.ibm.ws.http.options" ibm:type="pid"
-            type="String" required="false" default="defaultHttpOptions" />
+        <AD id="httpOptionsRef" name="%endpoint.httpOptions"
+	description="%endpoint.httpOptions.desc" ibm:reference="com.ibm.ws.http.options"
+	ibm:type="pid" type="String" required="false" default="defaultHttpOptions" />
```

Comment:
Can you clean up the formatting here?

Reply:
Formatting on this entire file looks fine on a text editor, but shows up messed up on the git preview. My guess is that there are 'tab' and whitespace characters that look fine in the eclipse or regular text editors but get messed up on this git view. I propose opening up a separate PR to clean up the entire file, since the formatting seems off for multiple other segments as well.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/18107